relayer/chains/cosmos/message_handlers.go

```diff
@@ -61,7 +61,7 @@ func (ccp *CosmosChainProcessor) handleChannelMessage(eventType string, ci provi
 		ccp.channelStateCache[channelKey] = false
 	case chantypes.EventTypeChannelOpenAck, chantypes.EventTypeChannelOpenConfirm:
 		ccp.channelStateCache[channelKey] = true
-	case chantypes.EventTypeChannelCloseInit, chantypes.EventTypeChannelCloseConfirm:
+	case chantypes.EventTypeChannelCloseConfirm:
```

Comment:
Is there a reason we don't want to consider the channel closed upon observing a committed `EventTypeChannelCloseInit`?

Reply:
`MsgTimeoutOnClose` may be a similar internal type that a module can use to cancel out pending packets upon a channel closure.

Reply:
The `MsgChannelCloseInit` is not included in this particular channel close flow (`MsgTimeout` on an ordered channel), but I'm not sure that it is unnecessary for all channel closures. We can remove it as it relates to the scope of this PR though. I created #868 to follow up